UI/ScorePoint/index.js

```diff
@@ -0,0 +1,11 @@
+import scorePointStyles from "../ScorePoint/scorePoint.module.css";
+import React from "react";
+
+export function ScorePoint({scorePointClassName}) {
```

Comment:
Is it worth adding a list of consts for what valid classNames might be? Something like this
https://github.com/gathercontent/gc-top-trumps/blob/5bd5a6022fc5cd696124fd5d5a6cd2e58b16f95c/UI/Icons/index.js#L18-L37

Then we can throw an error (or choose a default) if it's an invalid value?